File: gex/src/model/BarMatchAllyTeam.ts

export class BarMatchAllyTeam {
    public gameID: string = "";
    public allyTeamID: number = 0;
    public playerCount: number = 0;
    public won: boolean = false;
    public startBox: { top: number, bottom: number, left: number, right: number } = { top: 0, bottom: 0, left: 0, right: 0};

    public static parse(elem: any): BarMatchAllyTeam {
        return {
            ...elem
        };
    }

}

File: gex/Models/Db/BarMatchAllyTeam.cs
namespace gex.Models.Db {

    public class BarMatchAllyTeam {

        public string GameID { get; set; } = "";

        public int AllyTeamID { get; set; }

        public int PlayerCount { get; set; }

        public bool Won { get; set; }

        public Rectangle StartBox { get; set; } = Rectangle.Zero;

    }
}
